### PR TITLE
Add link to new CSI announce mailing list

### PR DESCRIPTION
**Pull request by saad-ali**
See https://groups.google.com/forum/#!topic/container-storage-interface-drivers-announce/IlwdtY23QP0

/release-note-none
/assign @msau42 
/assign @xing-yang

### CODE REVIEW

File: book/src/developing.md [hunk 1]
```diff
@@ -35,3 +35,7 @@ If your driver supports additional features, CSI "capabilities" can be used to a
 
 This is an partial list, please see the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md) for a complete list of capabilities.
 Also see the [Features](features.md) section to understand how a feature integrates with Kubernetes.
+
+## Remain Informed
```
Comment (msau42) [inline book/src/developing.md:39]: Maybe this should go first at the beginning? It can be easily skimmed if it's at the end

Comment (saad-ali) [reply book/src/developing.md:39]: Sure

**Comment by saad-ali** [comment]
PTAL

File: book/src/developing.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
```
Comment (msau42) [inline book/src/developing.md:1]: Fix indentation

Comment (saad-ali) [reply book/src/developing.md:3]: Whoops. Fixed.

**Comment by saad-ali** [comment]
PTAL

**Comment by xing-yang** [comment]
lgtm

**Pull request merged**